primitives/src/xvm.rs

```diff
@@ -64,8 +66,6 @@ pub enum CallError {
     InvalidTarget,
     /// Input is too large.
     InputTooLarge,
-    /// Bad origin.
-    BadOrigin,
     /// The call failed on EVM or WASM execution.
     ExecutionFailed(Vec<u8>),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `BadOrigin` variant has been removed from the `CallError` enum. Please ensure that this change doesn't affect any existing code or functionality.
```diff
     InvalidTarget,
     /// Input is too large.
     InputTooLarge,
-    /// Bad origin.
-    BadOrigin,
     /// The call failed on EVM or WASM execution.
     ExecutionFailed(Vec<u8>),
 }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->